ts-starter-complete-cotta/scripts/main.ts

```diff
@@ -1,4 +1,12 @@
-import { world, system, BlockPermutation, EntityInventoryComponent, ItemStack, DisplaySlotId } from "@minecraft/server";
+import {
+  world,
+  system,
+  BlockPermutation,
+  EntityInventoryComponent,
+  ItemStack,
+  DisplaySlotId,
+  ScoreboardIdentity,
```

Comment:
I think `ScoreboardIdentity` is not more needed

Reply:
You're absolutely right! I thought that was just a prettier fix thing. Good catch. Now it's a super tiny PR!